<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupDialog</class>
 <widget class="QDialog" name="SetupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="grbxSerialPort">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>221</width>
     <height>180</height>
    </rect>
   </property>
   <property name="title">
    <string>Serial port</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="cmbxSerialPort"/>
    </item>
    <item>
     <widget class="QLabel" name="lblSystemLocation">
      <property name="text">
       <string>System location</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblDescription">
      <property name="text">
       <string>Description</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblManufacturer">
      <property name="text">
       <string>Manufacturer</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblSerialNumber">
      <property name="text">
       <string>Serial number</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblVendorID">
      <property name="text">
       <string>Vendor ID</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblProductID">
      <property name="text">
       <string>Product ID</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
